src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/ChimericAlignment.java

```diff
@@ -179,12 +182,14 @@ public ChimericAlignment(final AlignmentInterval intervalWithLowerCoordOnContig,
                 }
             }
 
-            final SAMSequenceDictionary referenceDictionary = null; // don't need this for its intended cases, which are all same CHR mapping
-            final ChimericAlignment chimericAlignment = new ChimericAlignment(current, next, insertionMappings, alignedContig.contigName, referenceDictionary);
-            if (!chimericAlignment.isNotSimpleTranslocation())
-                throw new GATKException.ShouldNeverReachHereException("Mapped assembled contigs are sent down the wrong path: " +
-                        "contig suggesting \"translocation\" is sent down the insert/deletion path.\n" + alignedContig.toString());
-            results.add(chimericAlignment);
+            // TODO: 10/18/17 this way of filtering CA based on not quality but alignment characteristics is
+            // an artifact of development and relies on the dark secret that this was initially developped for calling ins/del,
```

Comment:
'developed'.

Also I'm not sure it needs to be portrayed as a an 'artifact' and a 'dark secret', there's nothing to be ashamed about.. just say that they are sent down different code paths right now.